src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -592,7 +568,7 @@ private async Task<ResourceConfig<VirtualMachineScaleSet>> SimpleParameterSetOrc
                     zones: _cmdlet.Zone,
                     ultraSSDEnabled: _cmdlet.EnableUltraSSD.IsPresent,
                     identity: _cmdlet.GetVmssIdentityFromArgs(),
-                    singlePlacementGroup: singlePlacementGroupFlexibleDefault,
+                    singlePlacementGroup: _cmdlet.SinglePlacementGroup.IsPresent,
```

Comment:
IsPresent could actually be true if the user passes in -SinglePlacementGroup:$false. So instead of using IsPresent, you should check if it is true.

Reply:
yeah for other changes below. It still is simple to invoke this function when  OrchestrationMode == null